routing/pathfind.go

```diff
@@ -26,6 +26,12 @@ const (
 
 	// infinity is used as a starting distance in our shortest path search.
 	infinity = math.MaxInt64
+
+	// RiskFactorBillionths controls the influence of time lock delta
```

Comment:
What was the process used to arrive at this value?

Reply:
Comment added